style.css

```diff
@@ -10,7 +10,8 @@ body {
 }
 .header {
   display: flex;
-  justify-content: space-between;  
+  justify-content: space-between;
+  margin-bottom: 100px;  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/HTML_CSS_2.0-Artem1998/pullRequest?prid=6242157)